docs/pbc/all/merchant-management/202410.0/base-shop/manage-in-the-back-office/create-merchants.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
This page was carried over from previous versions, with a new link added at the bottom.